generator/testdata/google/api/annotations.proto

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015 Google LLC
```

Comment:
I considered this option but I thought that it might be nice to just limit to the set of files we need so the input is clear. Happy to change this later on though